__test__/op-connect.test.ts

```diff
@@ -169,6 +170,54 @@ describe("Test OnePasswordConnect CRUD", () => {
         expect(vaults[0].name).toEqual(title);
         expect(vaults[1].name).toEqual(title);
     });
+
+    describe("getVaultByTitle", () => {
+
+        test("should throw an error if no vaults found", async () => {
+            const op = OnePasswordConnect(testOpts);
+            const title = "awsome_title";
```

Comment:
Noticed that these lines are repeated in this test group. Can we put them one level above? (i.e. in the same group test before any test is executed) It would reduce code duplication.

Reply:
done